### PR TITLE
chore(build): add Jenkinsfile to build via Jenkins

### PR DESCRIPTION
**Pull request by sagarkrsd**
This PR adds a Jenkinsfile which will enable this
repository to be built using Jenkins.

Signed-off-by: sagarkrsd <sagar.kumar@mayadata.io>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+			                         sh "docker push mayadataio/openebs-upgrade:${tag}"
+                            } else {
+			                   echo "WARNING: Not pushing Image"
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```
Comment (atulabhi) [inline Jenkinsfile]: Pipeline should be closed in last but it is cloning here. Remove this extra bracket

Comment (sagarkrsd) [reply Jenkinsfile]: Done

**Review by atulabhi — APPROVED** [review]
LGTM

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            }
+        }
+        stage('Push Image') {
+            steps {
+                script {
+		             withCredentials([usernamePassword( credentialsId: 'docke_cred', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+                            if (env.BRANCH_NAME == 'master')  {
+                               echo "Pushing the image with the tag..."
+                               sh "docker login -u${USERNAME} -p${PASSWORD} "
+			                         sh "docker push mayadataio/openebs-upgrade:${tag}"
```
Comment (atulabhi) [inline Jenkinsfile]: push should be same as docker build

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                            } else {
+			                   echo "WARNING: Not pushing Image"
+                        }
+                    }
+                }
+            }
+        }
+    }
+    /**
```
Comment (atulabhi) [inline Jenkinsfile]: please remove this comment

**Comment by AmitKumarDas** [comment]
@sagarkrsd Can we add more info in the PR e.g. what links will one refer to, the final image repo with tag & so on. How can we enable other teams to make use of the image & so on.
I would appreciate if there are comments in the Jenkinsfile as well.

**Review by AmitKumarDas — APPROVED** [review]
/lgtm

**Pull request merged**